sim/Cargo.toml

```diff
@@ -14,3 +14,4 @@ rand = "0.7.0"
 rand_xorshift = "0.2.0"
 serde = "1.0.98"
 serde_derive = "1.0.98"
+libm = "0.2.1"
```

Comment:
(no action, just fyi) We have to be careful that dependencies work everywhere, including wasm (because web support is pretty much just blocked on some text rendering support). Luckily, libm is pure Rust, specifically designed for wasm stuff. Nice!

Reply:
ok. i'll try to keep it at a minimum.